src/Windowing/Silk.NET.GLFW/Silk.NET.GLFW.csproj

```diff
@@ -5,12 +5,6 @@
       <RootNamespace>Silk.NET.GLFW</RootNamespace>
       <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     </PropertyGroup>
-
-    <ItemGroup>
```

Comment:
This might be needed, but we won't know until we have a testable prototype.

Reply:
I ended up changing the few `internal`s to public anyways, so this ended up being unnecessary.